src/views/BookmarkPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { TopNav, SideNav } from '../components';
+/**
+ * @description bookmark view page method
+ * @returns {HTMLDivElement} profile
+ */
+const Bookmark = () => (
+  <div>
+    <TopNav />
+    <section style={{ textAlign: 'center' }}>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
+      <h1>This is your BOOKMARK page!!</h1>
```

Comment:
@shaolinmkz you could have done this better by passing it as an array and using `.times` in lodash to multiple it to as much as you may want.

Reply:
On it..